### PR TITLE
Added zend-mvc v3 support

### PR DESCRIPTION
**Pull request by svycka**
Added zend-mvc v3 support and removed v2 support.
Removed zend-servicemanager v2, zend-eventmanager v2 and zend-stdlib v2 support. 
Removed PHP 5.5 support.

### CODE REVIEW

File: src/SlmLocale/Strategy/UriPathStrategy.php [hunk 1]
```diff
@@ -55,11 +54,11 @@ class UriPathStrategy extends AbstractStrategy
     protected $redirect_to_canonical;
     protected $sl;
     /**
-     * @var TreeRouteStack
+     * @var SimpleRouteStack
      */
     protected $router;
 
-    public function __construct(TreeRouteStack $router = null)
+    public function __construct(SimpleRouteStack $router = null)
```
Comment (svycka) [inline src/SlmLocale/Strategy/UriPathStrategy.php]: this possibly will fix https://github.com/basz/SlmLocale/issues/94 can you test @koseduhemak

**Comment by svycka** [comment]
Don't have a project with zf3 to test this at the moment, maybe someone can test?

**Comment by koseduhemak** [comment]
Why was the support for ZendFramework 2 removed? I currently have a ZF2 project and would like to use SlmLocale... Would it be possible to add support back for ZF2?

**Comment by svycka** [comment]
@koseduhemak we can't because of namespace change zend-mvc:2.0 was `Zend\Mvc\Router` and in v3 it is `Zend\Router` this is used in UriPathStrategy and localeUrl view helper if you could think a easy way to make them work then I guess it is possible. But not sure we need that because you can use 0.2.* for your zf2 app and update when you are ready.

I have updated my not so small app to v3 in a day or so and would have done that even faster but had to deal with old dependancies with was harder than upgrading zend packages

**Pull request merged**